docs/data/toolpad/connecting-to-data-sources/fetch.md

```diff
@@ -0,0 +1,32 @@
+# Fetch
+
+<p class="description">
+    <b>Fetch</b> query type is an easiest and fastest way to load external data within Toolpad app
+</p>
```

Comment:
A few changes: fix the missing leading dot, fix the formatting, remove it in the description, and fix a small grammar issue.

```suggestion
<p class="description">Fetch query type is the easiest and fastest way to load external data within Toolpad app.</p>
```

*x the other pages*